### ISSUE TITLE
Unable to get anime information using startp.sh

### ISSUE THREAD

**Issue by vivian-ng** [phanirithvij/twist.moe#29]
`startp.sh` seems to be having problems trying to get the information through the API.
```
$ ./startp.sh https://twist.moe/a/horimiya
making it https://twist.moe/a/horimiya/1
Fetching info...
rm: cannot remove '.temp.txt': No such file or directory
Traceback (most recent call last):
  File "../jsonP.py", line 5, in <module>
    sources = json.load(sys.stdin)
  File "/usr/lib/python3.8/json/__init__.py", line 293, in load
    return loads(fp.read(),
  File "/usr/lib/python3.8/json/__init__.py", line 357, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python3.8/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python3.8/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
Traceback (most recent call last):
  File "../decrypt.py", line 59, in <module>
    with open(file_name) as fn:
FileNotFoundError: [Errno 2] No such file or directory: '.temp.txt'
Creating horimiya/list.txt
copying download.sh to horimiya
Done
Check the Anime/horimiya folder
Enjoy
Done
```

For example, this seems to give an empty reply.
```
$ curl -s "https://twist.moe/api/anime/yuru-camp/sources" -H 'x-access-token: 1rj2vRtegS8Y60B3w3qNZm5T2Q0TN2NR'
<html>
<head><title>302 Found</title></head>
<body>
<center><h1>302 Found</h1></center>
<hr><center>nginx</center>
</body>
</html>
```

**Comment by FilipAndersson245** [comment]
the new access-token seem to be 0df14814b9e590a1f26d3071a4ed7974

**Comment by vivian-ng** [comment]
Should be fixed by this [PR#30](https://github.com/phanirithvij/twist.moe/pull/30). 
@phanirithvij Please review the PR, thank you.

**Comment by phanirithvij** [comment]
Closed by #30
